src/haddock/libs/libplots.py

```diff
@@ -281,3 +316,78 @@ def scatter_plots(capri_filename, cl_ranking):
             )
 
         fig.write_html(px_fname, full_html=False, include_plotlyjs='cdn')
+        
+
+def scatter_plots_png(capri_filename, cl_ranking, dpi):
+    """
+    Create scatter plots in png format.
+
+    analogous to scatter_plots, but here the plots are saved in png format.
```

Comment:
By looking at the functions `scatter_plots` and `scatter_plots_png` they indeed look very similar but it's unclear to me what's the actual difference between them; aka: why do you need a separate function for it?

Reply:
unfortunately the specifics of plotly and matplotlib make an effective integration very hard. will try to come out with something prettier!